plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineWizardPage.java

```diff
@@ -96,6 +88,13 @@ public void createControl(Composite parent) {
     Dialog.applyDialogFont(container);
   }
 
+  private void configureLibrariesSelectorGroup(Composite container) {
+    appEngineLibrariesSelectorGroup = new LibrarySelectorGroup(container, getSupportedLibraries(),
+        Messages.getString("app.engine.libraries.group")); //$NON-NLS-1$
+  }
+
+  protected abstract String getSupportedLibraries();
```

Comment:
Given this returns a `String`, I think it's good to have a Javadoc or change the name to `getSupportedLibrariesGroup`.

Reply:
changed the name